### PR TITLE
Add extension option for whether to hide other users starring/forking your repos

### LINKED ISSUE #193: Add some options
I noticed that you have an "against options" thought for this project. What's the reasoning for not adding options out of curiosity? Just so that it's a more simple interface? 

Main thing I want to disable/enable sometimes is that usually I want the "Hides other users starring/forking your repos from the newsfeed" feature enabled, but sometimes I want to quickly look at it (and then disable it afterwards again). If I added a button to do this would it be likely be accepted? Or do you just not want global options that isn't in the github UI?

### PR DESCRIPTION
**Pull request by bfred-it**
Oh boy.

I tried to keep it as contained as possible. Options are automatically saved.

Fixes #193 (Also #344 #174 #131 #75)

### CODE REVIEW

File: src/content.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import 'webext-dynamic-content-scripts';
+import OptSync from 'webext-options-sync';
```
Comment (sindresorhus) [inline src/content.js]: Awesome module!

File: src/options/index.js [hunk 2]
```diff
@@ -1,9 +1,16 @@
+import OptSync from 'webext-options-sync';
```
Comment (sindresorhus) [inline src/options/index.js]: Super nitpick, but `OptionsSync` would be more readable.

File: src/options/index.js [hunk 3]
```diff
@@ -1,9 +1,16 @@
...
+
+new OptSync().syncForm(document.querySelector('#options-form'));
```
Comment (sindresorhus) [inline src/options/index.js]: What do you think about `syncForm` accepting a CSS selector string for convenience?

**Comment by sindresorhus** [comment]
When I tried out this PR locally, the checkbox for the option was not checked, but according to the code, it should be `true` by default.

**Comment by sindresorhus** [comment]
Can you also add a note to the readme about us having some options and where to set them, so the user would know about it.

File: src/content.js (hunk 1 above)
Comment (bfred-it) [inline src/content.js]: Glad you like it! It's open to suggestions :)

File: src/options/index.js (hunk 3 above)
Comment (bfred-it) [inline src/options/index.js]: Will add

File: extension/options/index.css [hunk 4]
```diff
@@ -1,6 +1,8 @@
 html {
 	font-family: sans-serif;
 }
-input {
-	font: inherit;
+/* Increase spacing between sections */
+h2 ~ h2,
+form ~ form {
+	margin-top: 2em;
```
Comment (sindresorhus) [inline extension/options/index.css]: Test (ignore this)

**Comment by bfred-it** [comment]
✅

Hide other users starring/forking your repos option

**Pull request merged**